airflow/metadata.csv

```diff
@@ -10,7 +10,9 @@ airflow.ti_successes,count,,task,,Overall task instances successes,1,airflow,ti_
 airflow.zombies_killed,count,,task,,Zombie tasks killed,0,airflow,zombies_killed
 airflow.scheduler_heartbeat,count,,,,Scheduler heartbeats,0,airflow,scheduler_heartbeat
 airflow.dag_processing.processes,count,,,,Number of currently running DAG parsing processes,0,airflow,dag_proc.processes
-airflow.scheduler.tasks.killed_externally,gauge,,task,,Number of tasks killed externally,-1,airflow,scheduler.tasks.killed_externally
+airflow.scheduler.tasks.killed_externally,count,,task,,Number of tasks killed externally,-1,airflow,scheduler.tasks.killed_externally
```

Comment:
@AlexandreYang what do we do with this? (It was set as gauge but is a count)

Reply:
It's ok to fix the change. It's currently used with `.as_count()` in practice.

Reply:
as_count has no effect on gauges. But it's still ok to change because the type that matters in the end is the one sent by the agent, not on the metadata.csv